### PR TITLE
Some extra spelling suggestions for the dictionnary (aumatically, ...)

### PR DESCRIPTION
**Pull request by mdeweerd**
```
aumatically->automatically
eatch->each, teach,
sinal->signal
wrehouse->warehouse
wrehouses->warehouses
```

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -23278,6 +23279,7 @@ easly->easily
 easyer->easier
 easyest->easiest
 easyly->easily
+eatch->each, teach,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:23282]:
```suggestion
eatch->each, teach, watch,
```

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -8055,6 +8055,7 @@ auhtorized->authorized
 auhtorizes->authorizes
 auhtorizing->authorizing
 auhtors->authors
+aumatically->automatically
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:8058]:
```suggestion
aumatically->automatically, traumatically,
```

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -53333,6 +53335,7 @@ simutaneous->simultaneous
 simutaneously->simultaneously
 sinagog->synagogue
 sinagogs->synagogues
+sinal->signal
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:53338]:
```suggestion
sinal->signal, spinal,
```

Comment (mdeweerd) [reply codespell_lib/data/dictionary.txt:53338]: Added in new comit.

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (mdeweerd) [reply codespell_lib/data/dictionary.txt:8058]: Added in new 25a22a0

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (DimitriPapadopoulos) [reply codespell_lib/data/dictionary.txt:23282]: W and E are neighbours on QWERTY keyboards.

**Review by DimitriPapadopoulos — APPROVED** [review]

**Comment by DimitriPapadopoulos** [comment]
Thank you @mdeweerd.

**Comment by DimitriPapadopoulos** [comment]
Thank you @mdeweerd.

**Pull request merged**